autopub/__init__.py

```diff
@@ -0,0 +1,67 @@
+from pathlib import Path
```

Comment:
I'll move these classes into different files when we agree on the overall architecture :)